src/resources/views/crud/fields/select2_from_ajax_multiple.blade.php

```diff
@@ -2,7 +2,27 @@
 @php
     $connected_entity = new $field['model'];
     $connected_entity_key_name = $connected_entity->getKeyName();
-    $old_value = old(square_brackets_to_dots($field['name'])) ?? $field['value'] ?? $field['default'] ?? false;
+    $current_value = old(square_brackets_to_dots($field['name'])) ?? $field['value'] ?? $field['default'] ?? false;
+
+    $response_entity = isset($field['response_entity']) ? $field['response_entity'] :
```

Comment:
I would remove ```$response_entity``` entirely in this field type too, like I [mentioned here](https://github.com/Laravel-Backpack/CRUD/pull/2308/files#r370925597)